tests/eas/hmp_parity.py

```diff
@@ -400,108 +423,92 @@ def test_first_cpu_migrators(self):
-        big_residency_window = (self.offset + 5, self.offset + 10)
-
-        logging.info(
-            "Offload Migration and Idle Pull: Test Big Residency (Migrators)")
-
-        log_result(
-            self.m_assert.getResidency(
-                "cluster",
-                self.env.target.bl.bigs,
-                percent=True,
-                window=big_residency_window
-            ), self.log_fh)
-
-        self.assertTrue(
-            self.m_assert.assertResidency(
-                "cluster",
-                self.env.target.bl.bigs,
-                EXPECTED_RESIDENCY_PCT,
-                operator.ge,
-                percent=True,
-                window=big_residency_window,
-                rank=self.num_tasks),
-            msg="Not all 'migrator' tasks are running on big cores for at least {}% of their execution time"\
-                    .format(EXPECTED_RESIDENCY_PCT))
-
-
-    def test_migrators_switch(self):
-        "Offload Migration and Idle Pull: Test LITTLE -> BIG Idle Pull Switch (Migrators)"
-        switch_window = (self.offset + 4.5, self.offset + 5.5)
-
-        logging.info(
-            "Offload Migration and Idle Pull: Test LITTLE -> BIG Idle Pull Switch (Migrators)")
-        log_result(
-            self.m_assert.assertSwitch(
-                "cluster",
-                self.env.target.bl.littles,
-                self.env.target.bl.bigs,
-                window=switch_window), self.log_fh)
-
-        self.assertTrue(
-            self.m_assert.assertSwitch(
-                "cluster",
-                self.env.target.bl.littles,
-                self.env.target.bl.bigs,
-                window=switch_window,
-                rank=self.num_tasks),
-            msg="Not all 'migrator' tasks are pulled by idle big cores when expected")
-
-
-    def test_big_res_early_starters(self):
-        """Offload Migration and Idle Pull: Test Big Residency (EarlyStarters)"""
-        logging.info(
-            "Offload Migration and Idle Pull: Test Big Residency (EarlyStarters)")
-
-        big_residency_window = (self.offset, self.offset + 5)
-
-        log_result(self.e_assert.getResidency(
-            "cluster",
-            self.env.target.bl.bigs,
-            percent=True,
-            window=big_residency_window), self.log_fh)
-
-        self.assertTrue(
-            self.e_assert.assertResidency(
-                "cluster",
-                self.env.target.bl.bigs,
-                EXPECTED_RESIDENCY_PCT,
-                operator.ge,
-                percent=True,
-                window=big_residency_window,
-                rank=self.num_tasks),
-            msg="Not all 'early starter' tasks are running on big cores for at least {}% of their execution time"\
-                    .format(EXPECTED_RESIDENCY_PCT))
+    def test_big_cpus_fully_loaded(self):
+        """Offload Migration and Idle Pull: Big cpus are fully loaded as long as there are tasks left to run in the system"""
```

Comment:
nosetests only prints the first line of the docstring, so if you break the line it looks weird:
```
Offload Migration and Idle Pull: Big cpus are fully loaded as long as ... ok
```

I'd rather leave it like this, even if it breaks the 80 column rule.